SlicerConfiguration/SettingsControlSelectors.cs

```diff
@@ -258,18 +262,6 @@ private void MenuItem_Selected(object sender, EventArgs e)
 			// Ensure that activated or deactivated user overrides are always persisted to disk
 			activeSettings.Save();
 
-			UiThread.RunOnIdle(() =>
-			{
-				ApplicationController.Instance.ReloadAdvancedControlsPanel();
-				foreach (var keyName in PrinterSettings.KnownSettings)
-				{
-					if(settingBeforeChange[keyName] != ActiveSliceSettings.Instance.GetValue(keyName))
-					{
-						ActiveSliceSettings.OnSettingsChanged(SliceSettingsOrganizer.Instance.GetSettingsData(keyName));
-					}
-				}
-			});
-
```

Comment:
We specifically were ensuring that SettingsChanged events fired for stashed/popped item: https://github.com/MatterHackers/MatterControl/commit/83a3c875575ff507107979efdf6534b702c3921e

If we kill this, we need to change the code that stashes and pops to use SetValue and discover what the effects of that might have